pkgs/development/python-modules/kivy/default.nix

```diff
@@ -0,0 +1,101 @@
+{ stdenv, buildPythonPackage, fetchPypi, maintainers, platforms, licenses, pkgs, self }:
+
+# From the 1.10.0 documentation:
+# "This version of Kivy requires at least Cython version 0.23, and has been
+# tested through 0.25.2. Later versions may work, but as they have not been
+# tested there is no guarantee."
+
+assert stdenv.lib.versionAtLeast self.cython.version "0.23";
+
+# Rather than depending on a specific older version of Cython, we accept the
+# off-chance of breakage:
+# assert stdenv.lib.versionOlder self.cython.version "0.25.3";
+
+buildPythonPackage rec {
+  pname = "Kivy";
+  version = "1.10.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1394zh6kvf7k5d8vlzxcsfcailr3q59xwg9b1n7qaf25bvyq1h98";
+  };
+
+  # setup.py invokes git on build but we're fetching a tarball, so
+  # can't retrieve git version. We remove the git-invocation from setup.py
+  # This patch can presumably be removed on the next version of Kivy, because the problem
+  # was addressed in https://github.com/kivy/kivy/issues/5365
+  patches = [
+    ./setup.py.patch
+  ];
+
+  buildInputs = with self; [
+    pkgconfig
+    cython
```

Comment:
these two can be `nativeBuildInputs` if I am correct.

Reply:
I am not confident that I understand the difference (see https://groups.google.com/forum/#!topic/nix-devel/Ad1b0RShe6k) and all other Python packages that I could find do in fact use `buildInputs` here, so I'm a bit wary to proceed with this tip. Feel free to correct me if you're sure about this behavior though.